igor-core/src/main/java/com/netflix/spinnaker/igor/build/model/GenericGitRevision.java

```diff
@@ -36,4 +36,20 @@
   private String message;
   private Instant timestamp;
   private String remoteUrl;
+
+  public GenericGitRevision withAuthorInfo(String committer, String message) {
+    this.committer = committer;
+    this.message = message;
+    return this;
+  }
+
+  public GenericGitRevision withTimestamp(Instant timestamp) {
+    this.timestamp = timestamp;
+    return this;
+  }
+
+  public GenericGitRevision withLink(String link) {
```

Comment:
Consider renaming for consistency with what the method is changing:
```suggestion
  public GenericGitRevision withCompareUrl(String link) {
```

Reply:
fixed